osscameroon/camerdevs
# feat: add companies GET endpoint

This pull request add the GET companies endpoint

Steps to verify:
- move to the `backend` folder with `cd ./backend`
- run `make start-postrges` to run an initialized database
- run  `make run` to launch the api
- then run the command `curl -X GET localhost:7000/companies` you should see something like this
```
{"id":1,"name":"Brainverse","rating":2,"createdat":"0001-01-01T00:00:00Z","updatedat":"0001-01-01T00:00:00Z"},{"id":2,"name":"Photofeed","rating":4,"createdat":"0001-01-01T00:00:00Z","updatedat":"0001-01-01T00:00:00Z"},
....
]
```
